libraries/libfc/src/crypto/bls_public_key.cpp

```diff
@@ -50,24 +21,17 @@ namespace fc { namespace crypto { namespace blslib {
    :_pkey(parse_base58(base58str))
    {}
 
-
-   bool bls_public_key::valid()const
-   {
-      //return std::visit(is_valid_visitor(), _storage);
-   }
-
-
    std::string bls_public_key::to_string(const fc::yield_function_t& yield)const {
 
       std::vector<char> v2;
       std::array<uint8_t, 48> bytes = _pkey.toCompressedBytesBE();
       std::copy(bytes.begin(), bytes.end(), std::back_inserter(v2));
 
       std::string data_str = fc::to_base58(v2, yield);
-
-      //std::string data_str = Util::HexStr(_pkey);
       
-      return std::string(config::bls_public_key_base_prefix) + "_" + data_str;
+      //return std::string(config::bls_public_key_base_prefix) + "_" + data_str;

```

Comment:
Seems like we would want to keep a prefix. @arhag ?

Reply:
Yes. See https://github.com/AntelopeIO/leap/pull/1538#discussion_r1301981962.